blockdata/chainmonitor.go

```diff
@@ -56,21 +47,21 @@ func (p *chainMonitor) collect(hash *chainhash.Hash) (*wire.MsgBlock, *BlockData
 	height := block.Height()
 	log.Infof("Block height %v connected. Collecting data...", height)
 
-	if len(p.watchaddrs) > 0 {
-		// txsForOutpoints := blockConsumesOutpointWithAddresses(block, p.watchaddrs,
-		// 	p.collector.dcrdChainSvr)
-		// if len(txsForOutpoints) > 0 {
-		// 	p.spendTxBlockChan <- &BlockWatchedTx{height, txsForOutpoints}
-		// }
-
-		txsForAddrs := txhelpers.BlockReceivesToAddresses(block,
-			p.watchaddrs, p.collector.netParams)
-		if len(txsForAddrs) > 0 {
-			p.recvTxBlockChan <- &txhelpers.BlockWatchedTx{
-				BlockHeight:   height,
-				TxsForAddress: txsForAddrs}
-		}
-	}
+	// if len(p.watchaddrs) > 0 {
+	// 	// txsForOutpoints := blockConsumesOutpointWithAddresses(block, p.watchaddrs,
+	// 	// 	p.collector.dcrdChainSvr)
+	// 	// if len(txsForOutpoints) > 0 {
+	// 	// 	p.spendTxBlockChan <- &BlockWatchedTx{height, txsForOutpoints}
+	// 	// }
+	//
+	// 	txsForAddrs := txhelpers.BlockReceivesToAddresses(block,
+	// 		p.watchaddrs, p.collector.netParams)
+	// 	if len(txsForAddrs) > 0 {
+	// 		p.recvTxBlockChan <- &txhelpers.BlockWatchedTx{
+	// 			BlockHeight:   height,
+	// 			TxsForAddress: txsForAddrs}
+	// 	}
+	// }
```

Comment:
I left this code, because most of these methods and types are only referenced here, and I wanted to gather feedback before I nuked them. In particular, I think @chappjc had mentioned possibly re-implementing something around this area.

Reply:
Nuke it!